standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionInfo.java

```diff
@@ -79,8 +79,8 @@ public class CompactionInfo implements Comparable<CompactionInfo> {
   public Set<Long> writeIds;
   public boolean hasUncompactedAborts;
 
-  byte[] metaInfo;
-  String hadoopJobId;
+  public byte[] metaInfo;
+  public String hadoopJobId;
```

Comment:
looks like it's not in toString()

Reply:
I think not, with @deniskuzZ  we discussed that CompactionInfo should not be touched as a part of this PR.